canonical-web-and-design/jaas-dashboard
# Add Framer Motion and add little animation to Login box

## Done

Add Framer Motion and add little animation to Login box

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Refresh page and verify little login zoom animation
